softdevice_controller/CHANGELOG.rst

```diff
@@ -9,6 +9,23 @@ Changelog
 
 All the notable changes to this project are documented on this page.
 
+nRF Connect SDK v2.5.2
+**********************
+
+All the notable changes included in the |NCS| v2.5.2 release are documented in this section.
+
+Bug fixes
+=========
+
+* Fixed an issue where the controller stopped generating advertising reports.
+  This could happen when the controller was running an extended cooperative scanner together with other activities, such as advertising or connection,
+  while receiving data in an extended advertising event that used ``AUX_CHAIN_IND``. (DRGN-21020)
```

Comment:
```suggestion
  This could happen when the controller was running an extended cooperative scanner together with other activities, such as advertising or connection, while receiving data in an extended advertising event that used ``AUX_CHAIN_IND`` (DRGN-21020).
```
Amended Pekka's suggestion.